advanced/subpath/route53-cloudfront.mdx

```diff
@@ -158,6 +158,29 @@ Lastly, we're going to edit the `Default (*)` behavior.
 
 2. Select **Save changes**.
 
+### Additional behaviors for layout stability
+
+To prevent layout shifts and ensure smooth navigation, add these additional behaviors for static asset handling:
```

Comment:
Are these steps only applicable to some users or should everyone set these behaviors?

If everyone should do it, I'd remove this section and raise all the H4s to H3s so that it doesn't seem like these are extra optional settings.